src/town_cmd.cpp

```diff
@@ -2272,6 +2272,14 @@ bool GenerateTowns(TownLayout layout)
 	return false;  // we are still without a town? we failed, simply
 }
 
+void ExpandTowns()
+{
+	for (Town *t : Town::Iterate()) {
+		Command<CMD_EXPAND_TOWN>::Do(DC_EXEC, t->index, 0);
+	}
+
+	UpdateAirportsNoise();
```

Comment:
Why is this here? (and only this?) I'd expect noise levels to be handled by each expand-town cmd

Reply:
I looked at similar code paths and thought it was necessary, and only done after all towns have been expanded. Fixed it!